web-console/4.0-designs/overview/overview.md

```diff
@@ -0,0 +1,87 @@
+  * The default route will be set only for the current user.
+
+
+
+## Side panel view
+
+### General
+
+* When a row is clicked, the side panel will open with a summary for that resource.
+* The object badge and name are displayed at the top of the panel, along with an actions dropdown for users to take action on the object.
+* The side panel has two main tabs: `Overview` and `Resources`.
+* The `Overview` tab is the default view, and the list view content is pushed to the left.
+* The selected row remains highlighted, and the metrics column is hidden.
+* When necessary, the resource name, notifications, and route URL will wrap.
+* The `Group by label` dropdown and filter bar are not accessible when the side panel is open.
+* Users can close the side panel by clicking the 'x' in the top right corner of the panel or hitting 'Escape' on their keyboard.
+  * If a user closes the side panel, it will always reopen to the `Overview` tab.
+  * If a user changes which resource they are viewing with the side panel open, the current tab should remain selected for the new resource.
+  * If a user changes to the topology view with the side panel open, the panel will remain open on the same object.
+
+### Overview tab
+
+![overview](img/overview.png)
+* Each section is separated by a horizontal separator.
+* The first section has metrics and pod information.
+* The second section matches the details page view, including data such as the resource name, namespace, and labels.
+* The third section displays the resource's container information.
+
+**Pod Visualization**
+
+![pod](img/overview2.png)
+* The pod visualization is the same animation that occurs on a resource's full overview page, but scaled down to fit in the side panel. While the pod animation is occurring, the metrics graph is hidden from view.
+* Users can scale pods by using the up and down arrows next to the pod donut.
+  * If the side panel is closed while a pod is scaling, the animation will be displayed in the pods column in the list view.
+
+### Resources tab
+
+![resources](img/resources.png)
+
+**Builds section**
+
+* The first section provides information about the resource's builds.
+* Users may redirect to a build overview by clicking the build number.
+  * An icon to the left of the build number will indicate the build status.
+  * A relative timestamp will be displayed after the status.
+  * A `View Log` link will be available to access the full log for every build.
+* The builds will be listed in order of recency and will always show at least the one latest build.
+* Users may see a log snippet of any currently running builds.
+* Running builds will always be shown, unless there are more than three running at one time. In this case, only the three most recent builds will be displayed.
```

Comment:
Note that there are multiple states (new, pending, running). I think we want to make sure to show the one currently running even if there are more than 3 other builds in new/pending.

Reply:
@spadgett  Does your comment imply there can only be one technically "running" at one time? I may be misunderstanding.

Reply:
It depends on what's set in the build config. You can set up serial builds so only one runs at a time and newer builds gets queued up in new or pending states. Or you can set up parallel builds that can have any number running.

It's possible with serial builds to have several in new state and an older one running. Just commenting that the running build should display in that case.

Reply:
Got it, thanks for the clarification! @bmignano let's make sure we have this clear in the docs 👍